ansible_runner/utils.py

```diff
@@ -326,6 +306,19 @@ def _emit_event(self, buffered_stdout, next_event_data=None):
             event_data['end_line'] = self._start_line + n_lines
             self._start_line += n_lines
             if self._event_callback:
+                if event_data.get('stdout'):
+                    if self.output_json:
+                        stdout_actual = json.dumps(event_data)
```

Comment:
In the case that `output_json` is true, do you need to skip the case of `stdout_actual != "{}"`? It seems that that's a bit of logic that may have been missed. But then again, events with blank lines are extremely common, and I have no idea what the expectations could possibly be for those with that setting.